src/modules/navigator/rtl.cpp

```diff
@@ -223,9 +244,8 @@ RTL::set_rtl_item()
 
 		_mission_item.lat = _navigator->get_home_position()->lat;
 		_mission_item.lon = _navigator->get_home_position()->lon;
-		_mission_item.altitude_is_relative = false;
-		_mission_item.altitude = _navigator->get_home_position()->alt + _param_descend_alt.get();
-		_mission_item.yaw = _navigator->get_home_position()->yaw;
+		// don't change altitude
+		_mission_item.yaw = NAN; // don't bother with yaw
```

Comment:
I'm not sure I want this change - I want systems to have the same yaw as they took off with at the point of landing, because that is what the pilot needs to take over manually during landing. So I would not want to accept this change as-is.